test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu
+
+# folder we create the code in + cleanup
+tmp=$(mktemp -d)
+trap "rm -fr $tmp" 0 2 3 15
+
+for yaml in library/*/*/template.yaml
```

Comment:
I believe idea is to generate yamls from rego, we might not need to go through yaml at all cc @ctab

Reply:
there are no .rego files here, so would prefer to get this merged as is, separate PR can extract regos/deal with template building

Reply:
Hi @grosser,
Sorry for the confusion as we have been restructuring this repo. We do already have rego test files (and rego source files) for the policies in this repo in the `src/` directory. Example: `https://github.com/open-policy-agent/gatekeeper-library/blob/master/src/general/requiredlabels/src_test.rego`
Can you modify this script to use the existing rego source and test files?